src/options/lines-between-rulesets.js

```diff
@@ -0,0 +1,145 @@
+'use strict';
+
+let gonzales = require('gonzales-pe');
+
+let option = {
+  newLines: '',
+  space: null,
+
+  /**
+   * Option's name as it's used in config.
+   * @type {String}
+   */
+  get name() {
+    return 'lines-between-rulesets';
+  },
+
+  /**
+   * Name of option that must run after this option.
+   * @type {String}
+   */
+  get runBefore() {
+    return 'block-indent';
+  },
+
+  /**
+   * List of syntaxes that are supported by this option.
+   * @type {Array}
+   */
+  get syntax() {
+    return ['scss', 'css', 'less'];
+  },
+
+  /**
+   * Types of values this option accepts in config.
+   * @type {Object}
+   */
+  get accepts() {
+    return {
+      number: true
+    };
+  },
+
+  /**
+   * Sets the value of this option.
+   * @type {Number}
+   */
+  setValue(value) {
+    if (typeof value === 'number') {
+      value = new Array(Math.round(value) + 2).join('\n');
+    } else {
+      let err = 'This option only accepts numbers';
+      err += ', but you provided %s';
+
+      throw new Error(err, value);
+    }
+
+    return value;
+  },
+
+  /**
+   * Processes ast and fixes found code style errors.
+   * @param {Node} ast
+   */
+  process(ast) {
+    this.newLines = this.value;
+    this.space = gonzales.createNode({type: 'space', content: this.newLines});
+    this.processBlock(ast);
+  },
+
+  processBlock(x) {
+    let that = this;
+
+    if (x.is('stylesheet')) {
+      // Check all @rules
+      this.findAtRules(x);
```

Comment:
Better name this function as `processAtRules` or some other name, as it does more than just finding at rules.